Aptabase.uplugin

```diff
@@ -1,6 +1,6 @@
-﻿{
+{
 	"FileVersion": 3,
-	"VersionName": "0.0.1",
+	"VersionName": "0",
```

Comment:
I've set this to `0`, let me know if you think a better default would be helpful.

The actual value is now properly set by tags in `plugin-cd.yml` via https://github.com/outoftheboxplugins/BuildMachines/blob/master/.github/workflows/release-plugin.yml